File: solutions/2025-dec/25/snowflake_generator.js
function generateSnowflake(crystals) {
    const snows = crystals.split("\n");
    let ans = "";
    for (let i = 0; i < snows.length; i++) {
        const snow = snows[i];
        let snow_copy = "";
        for (let j = snow.length - 1; j >= 0; j--) {
            snow_copy += snow[j];
        }
        ans += snow;
        ans += snow_copy;
        if (i != snows.length - 1) ans += "\n";
    }
    return ans;
}

File: solutions/2025-dec/25/snowflake_generator.py
def generate_snowflake(crystals):
    snows = crystals.split("\n")
    ans = ""
    for i in range(len(snows)):
        snow = snows[i]
        snow_copy = snow[::-1]
        ans += snow
        ans += snow_copy
        if i != len(snows) - 1:
            ans += "\n"
    return ans
